app/src/main/java/edu/cascadia/mobas/pollinatorpathway/ui/login/PnwppDb.java

```diff
@@ -0,0 +1,44 @@
+package edu.cascadia.mobas.pollinatorpathway.ui.login;
+
+import android.content.Context;
+
+import androidx.room.Database;
+import androidx.room.Room;
+import androidx.room.RoomDatabase;
+
+@Database(entities = {Profile.class, Planting.class, Box.class}, version = 1)
+public abstract class PnwppDb extends RoomDatabase {
+
+    private static final String DATABASE_NAME = "user.db";
+
+    private static PnwppDb mPnwppDatabase;
+
+
+    //Singleton
+    public static PnwppDb getInstance(Context context){
+        if (mPnwppDatabase == null){
+            mPnwppDatabase = Room.databaseBuilder(context, PnwppDb.class, DATABASE_NAME)
+                    .allowMainThreadQueries()
+                    .build();
+            mPnwppDatabase.addStarterData();
+        }
+        return mPnwppDatabase;
+    }
+
+
+
+    public abstract ProfileDao profileDao();
+    public abstract PlantingDao plantingDao();
+    public abstract BoxDao boxDao();
+
```

Comment:
You will likely want to delete the StartedData before adding it back in, otherwise you'll get duplicate records each time you run the app.

Take a look at the Room with a View WordRoomDatabase to see some sample code for this